content/en/news/releases/1.17.x/announcing-1.17/change-notes/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Istio 1.17.0 Change Notes
+linktitle: 1.17.0
+subtitle: Minor Release
+description: Istio 1.17.0 change notes.
+publishdate: 2023-02-14
+release: 1.17.0
+weight: 10
+---
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/docs/releases/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+- **Deprecated** setting `PILOT_CERT_PROVIDER` to `kubernetes` for Kubernetes versions less than 1.20. [PR](https://github.com/istio/istio/pull/42233)
+
```

Comment:
We seem to be inconsistent with using Issue and PR #'s in the link text. It seems we include the issue number in the link text, but not the PR. At least in the case of no issue and a PR, we should have the # similar to having it for the issue alone.

Also, sometimes the issue is surrounded with parends (at least one case where there is only one parent) and then other times not.

Reply:
It is inconsistent right now. This is the last bit of cleanup I need to do. Hopefully, in the next couple of commits.

Reply:
Reviews again and changes required below, so resolving this section.